experiments/conductor/cmd/runner/prompts/mockgcp-generate-script.txt

```diff
@@ -0,0 +1,65 @@
+I am trying to create a test case for mockgcp.
```

Comment:
Personally I would rather organize by function rather than type. Having all the mockgcp components located together makes more sense to me than keeping all the prompts located together.

Reply:
I tried grouping by function and it's going to be a big PR.  I'll start chipping away at it.

For now, I moved them to be together, but it means they are all in the monopackage